src/crypto/tls/handshake_server_tls13.go

```diff
@@ -892,6 +942,11 @@ func (hs *serverHandshakeStateTLS13) sendServerParameters() error {
 }
 
 func (hs *serverHandshakeStateTLS13) requestClientCert() bool {
+	for _, flag := range hs.tlsFlags {
+		if flag == ExperimentalFlagSupportMTLS {
+			return true
+		}
+	}
```

Comment:
I'd suggest adding a test that demonstrates request mTLS when triggered by this flag.